# HG changeset patch
# User Steve Fink <sfink@mozilla.com>
# Date 1456209760 28800
#      Mon Feb 22 22:42:40 2016 -0800
# Node ID 25697d5038158fadb93ec3ed76f3c2e84fd4f60f
# Parent  7cdb490d72b5682c0b30bec2f716346e2029f52d
Backed out changeset 7cdb490d72b5 (bug 1250287) on a CLOSED TREE

diff --git a/js/src/devtools/rootAnalysis/build/gcc-b2g.manifest b/js/src/devtools/rootAnalysis/build/gcc-b2g.manifest
--- a/js/src/devtools/rootAnalysis/build/gcc-b2g.manifest
+++ b/js/src/devtools/rootAnalysis/build/gcc-b2g.manifest
@@ -1,11 +1,12 @@
 [
 {
-"version": "gcc 4.9.3",
-"size": 102421980,
-"visibility": "public",
-"digest": "f25292aa93dc449e0472eee511c0ac15b5f1a4272ab76cf53ce5d20dc57f29e83da49ae1a9d9e994192647f75e13ae60f75ba2ac3cb9d26d5f5d6cabf88de921",
+"gcc_version": "4.7.2"
+},
+{
+"size": 24657188,
+"digest": "28258eff929158c04dd7d6839bd3c4aa8963c2d478fe88e55cc98d58d878cc673e3625be5c92fe5b6c45bae38980b9919d9aaa7f32a2d8fee89294e90e2a1e2f",
 "algorithm": "sha512",
 "filename": "gcc.tar.xz",
 "unpack": true
 }
 ]
diff --git a/js/src/devtools/rootAnalysis/build/gcc.manifest b/js/src/devtools/rootAnalysis/build/gcc.manifest
--- a/js/src/devtools/rootAnalysis/build/gcc.manifest
+++ b/js/src/devtools/rootAnalysis/build/gcc.manifest
@@ -1,14 +1,15 @@
 [
 {
-"version": "gcc 4.9.3",
-"size": 102421980,
-"visibility": "public",
-"digest": "f25292aa93dc449e0472eee511c0ac15b5f1a4272ab76cf53ce5d20dc57f29e83da49ae1a9d9e994192647f75e13ae60f75ba2ac3cb9d26d5f5d6cabf88de921",
+"gcc_version": "4.7.2"
+},
+{
+"size": 79831648,
+"digest": "958d0dfb531ac6911187cf57907317144a547b884c79dcb483668f0e468ab9f031492ab49177edebf6a6437680b69f54a346a37b4316da78c0ff87aa39c5f2c3",
 "algorithm": "sha512",
 "filename": "gcc.tar.xz",
 "unpack": true
 },
 {
 "size": 12072532,
 "digest": "3915f8ec396c56a8a92e6f9695b70f09ce9d1582359d1258e37e3fd43a143bc974410e4cfc27f500e095f34a8956206e0ebf799b7287f0f38def0d5e34ed71c9",
 "algorithm": "sha512",
diff --git a/js/src/devtools/rootAnalysis/build/sixgill-b2g.manifest b/js/src/devtools/rootAnalysis/build/sixgill-b2g.manifest
--- a/js/src/devtools/rootAnalysis/build/sixgill-b2g.manifest
+++ b/js/src/devtools/rootAnalysis/build/sixgill-b2g.manifest
@@ -1,10 +1,13 @@
 [
-{
-"hg_id" : "ec7b7d2442e8",
-"algorithm" : "sha512",
-"digest" : "49627d734df52cb9e7319733da5a6be1812b9373355dc300ee5600b431122570e00d380d50c7c5b5003c462c2c2cb022494b42c4ad00f8eba01c2259cbe6e502",
-"filename" : "sixgill.tar.xz",
-"size" : 2628868,
-"unpack" : true
-}
+   {
+      "hg_id" : "0b889f399d47+ cleaner/qtip/tip",
+      "sixgill_version" : "0.9.1-177"
+   },
+   {
+      "algorithm" : "sha512",
+      "digest" : "bb7fcbcaf8d160bba92fa66b2e3607f3335d68c8339996c880ea5757c6e611b74ced6c6d5c50d523fc8d688cc922ee6e1628f3fb514af8ebeb9c877ad580e5fd",
+      "filename" : "sixgill.tar.xz",
+      "size" : 2159496,
+      "unpack": true
+   }
 ]
